Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apt-get update && apt-get -y install vim
 
 RUN cd $GOPATH/src && mkdir -p metricfunc
 COPY . $GOPATH/src/metricfunc
-RUN cd $GOPATH/src/metricfunc/cmd/metricfunc && CGO_ENABLED=0 go build
+RUN cd $GOPATH/src/metricfunc/cmd/metricfunc && CGO_ENABLED=0 go build -mod=mod
```

Comment:
Can you please tell why this flag ? I did not get enough details about this flag in "go build help".

Reply:
Here, -mod=mod tells the go command to ignore the vendor directory and to automatically update go.mod, for example, when an imported package is not provided by any known module.

If we do not use  this flag docker build fails with the following error. To fix this we require this flag.

```
 > [builder 5/5] RUN cd /go/src/metricfunc/cmd/metricfunc && CGO_ENABLED=0 go build:                                                                                                   
#14 0.432 go: inconsistent vendoring in /go/src/metricfunc:                                                                                                                            
#14 0.432       github.com/omec-project/metricfunc: is marked as replaced in vendor/modules.txt, but not replaced in go.mod                                                            
#14 0.432                                                                                                                                                                              
#14 0.432 	To ignore the vendor directory, use -mod=readonly or -mod=mod.
#14 0.432 	To sync the vendor directory, run:
#14 0.432 		go mod vendor
------
executor failed running [/bin/sh -c cd $GOPATH/src/metricfunc/cmd/metricfunc && CGO_ENABLED=0 go build]: exit code: 1
Makefile:24: recipe for target 'docker-build' failed
make: *** [docker-build] Error 1;
```